src/test/java/reposense/model/RunConfigurationDeciderTest.java

```diff
@@ -70,14 +70,14 @@ public void parse_addRepos_returnsCliRunConfiguration() throws Exception {
     }
 
     @Test
-    public void parse_reposAndConfigNotSpecified_returnsConfigRunConfiguration() throws Exception {
+    public void parse_reposAndConfigNotSpecified_doesNotReturnCliRunConfiguration() throws Exception {
         String input = new InputBuilder()
                 .addSinceDate("01/07/2017")
                 .addUntilDate("30/11/2017")
                 .addView()
                 .build();
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
         RunConfiguration runConfiguration = getRunConfiguration(cliArguments);
-        Assertions.assertTrue(runConfiguration instanceof ConfigRunConfiguration);
+        Assertions.assertFalse(runConfiguration instanceof CliRunConfiguration);
     }
 }
```

Comment:
possibly add a test for OneStopConfigRunConfiguration

Reply:
Test case added.